install-dnanexus-cli.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 
+set -e -o pipefail -u
+
+DX_TOOLKIT_VERSION=0.285.0
+DX_UPLOAD_AGENT_VERSION=1.5.31
+
 # get the dx-toolkit
-curl -s https://wiki.dnanexus.com/images/files/dx-toolkit-v0.255.0-ubuntu-14.04-amd64.tar.gz | tar -xzp
+curl -s https://wiki.dnanexus.com/images/files/dx-toolkit-v${DX_TOOLKIT_VERSION}-ubuntu-14.04-amd64.tar.gz | tar -xzp
```

Comment:
It [looks like](https://documentation.dnanexus.com/downloads) DNAnexus is moving away from a wiki for their docs, and is also moving their downloads to a new S3-backed source, ex.:
https://dnanexus-sdk.s3.amazonaws.com/dx-toolkit-v0.285.0-ubuntu-16.04-amd64.tar.gz

The old URL works for now, but maybe we should switch to the newer one.

Reply:
fixed